tests/testsuite/support/install.rs

```diff
@@ -23,9 +24,5 @@ pub fn cargo_home() -> PathBuf {
 }
 
 pub fn exe(name: &str) -> String {
-    if cfg!(windows) {
-        format!("{}.exe", name)
-    } else {
-        name.to_string()
-    }
+    format!("{}{}", name, EXE_SUFFIX)
```

Comment:
Why `format!` rather than the more performant `[name, EXE_SUFFIX].concat()`?

Reply:
I have nothing again using that instead.